TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/TeleOpMovement.kt

```diff
@@ -42,6 +43,12 @@ object TeleOpMovement : Component {
         val joyMagnitude = sqrt(joyY * joyY + joyX * joyX)
 
         // all movement of the wheels
-        TriWheels.driveWithRotation(joyRadians, joyMagnitude, rotationPower)
+        // if the right bumper is pressed move slower
+        if (gamepad.right_bumper) {
+            TriWheels.driveWithRotation(joyRadians, joyMagnitude * SLOW_MODE, rotationPower)
+        } else {
+            // normal movement of all wheels
+            TriWheels.driveWithRotation(joyRadians, joyMagnitude, rotationPower)
```

Comment:
Is there a way that we don't have do duplicate the `driveWithRotation` line? In case we change add something in the future, we shouldn't have to change two different places.